cs3api4lab/config/config_manager.py

```diff
@@ -37,8 +37,16 @@ def __init__(self):
 
         self.config = self.config | config
 
-        # overwriting the values with env vars
-        env_vars = {env[4:].lower():os.environ[env] for env in os.environ if env.startswith("CS3_")}
+        # overwriting the values with env vars and parse boolean values
+        env_vars = {}
+        for env in os.environ:
+            if env.startswith("CS3_"):
+                if env[4:].lower() in ["secure_channel", "tus_enabled", "enable_ocm"]:
+                    value = os.environ[env] == "true"
```

Comment:
Why don't you parse the string to a boolean? Otherwise, `CS3_SECURE_CHANNEL=false` will be set as true...

Reply:
I'm not sure it wouldn't be better to just do #148 instead of this. It shouldn't take much time to do it and we just get it done.

Reply:
I will take a look then

Reply:
@diocas we will look at the https://github.com/sciencemesh/cs3api4lab/issues/148 a little bit later if that's ok, right now we want to focus on the higher priority tasks. https://github.com/sciencemesh/cs3api4lab/issues/148 is not as quick as it seems, there are a lot of places where config is used plus tests.

Reply:
> @diocas we will look at the #148 a little bit later if that's ok, right now we want to focus on the higher priority tasks. #148 is not as quick as it seems, there are a lot of places where config is used plus tests.

Ok, no problem. I thought it was just a matter of replacing some things.

Reply:
> Why is that? Any string other than 'true' should be evaluated as false

Ah sorry!! I misread the code!!